dojo/models.py

```diff
@@ -832,11 +832,11 @@ def get_response(self):
 
 class Risk_Acceptance(models.Model):
     path = models.FileField(upload_to='risk/%Y/%m/%d',
-                            editable=False, null=False,
+                            editable=True, null=False,
```

Comment:
Non-editable fields cannot be included in modelforms. Will try making them read-only in their respective modelforms. Is this the right approach?